charts/prometheus-statsd-exporter/values.yaml

```
@@ -0,0 +1,118 @@
+# Default values for prometheus-statsd-exporter.
+# This is a YAML-formatted file.
+# Declare variables to be passed into your templates.
+
+replicaCount: 1
+# deploymentRevisionHistoryLimit: 10
+
+image:
+  repository: prom/statsd-exporter
+  pullPolicy: IfNotPresent
+  # Overrides the image tag whose default is the chart appVersion.
+  tag: "latest"
```

Default tag should be pinned.
```suggestion
  tag: v0.18.0
```